content/articles/collapsing-action-tool-bar-with-android-material-design-in-kotlin/index.md

```diff
@@ -0,0 +1,261 @@
+### Introduction
+Collapsing Toolbar Layout widget is used in conjunction with the basic toolbar widget to create a more sophisticated version of the regular toolbar. Expanding and collapsing toolbar on-screen scrolling is a fantastic toolbar motion made possible by this widget. This feature will automatically increase the toolbar size set in the App Bar Layout widget.
+
+This tutorial will build an application that utilizes the Collapsing Toolbar layout widget. We shall also include Android's new design support library to create scrolling effects that are eye-catching animations(palette API). There will be precise step-by-step procedures that will be easy to follow and understand for beginners and intermediate.
+
+### Table of contents
+- [Setting up the project environment](#setting-up-the-project-environment)
+- [Adding the appropriate dependencies](#adding-the-appropriate-dependencies)
+- [Designing the XML layout](#designing-the-xml-layout)
+- [Writing Kotlin code](#writing-kotlin-code)
+- [Testing the application](#testing-the-application)
+- [Conclusion](#conclusion)
+
```

Comment:
Add a Prerequisites' section here, to provide the audience with any prerequisites that may be required to follow your tutorial. For example: Need basic HTML and CSS knowledge for an article on how to use the CSS pre-processor, Sass.